src/client/containers/BoardPage/BoardPageComponents/CandidateList/CandidateListPostDeadline.js

```diff
@@ -4,12 +4,8 @@ import { CardItemDecorator } from '../../../../components/CandidateCard/Candidat
 import { SortableItem } from '../../../DragAndSortAdapter/DragAndSortAdapter';
 import PropTypes from 'prop-types';
 
-export default function CandidateListPostDeadline({
-  userId,
-  boardId,
-  displayDelete,
-}) {
-  const { candidates, error } = useCandidates(userId, boardId);
+export default function CandidateListPostDeadline({ displayDelete }) {
```

Comment:
You are still showing the error here. In the other pr you removed it after Slavias review